cmd/uplink/cmd/mb.go

```diff
@@ -45,7 +46,7 @@ func makeBucket(cmd *cobra.Command, args []string) error {
 
 	project, err := cfg.GetProject(ctx)
 	if err != nil {
-		return fmt.Errorf("error setting up project: %+v", err)
+		return errs.Wrap(fmt.Errorf("error setting up project: %+v", err))
```

Comment:
Use `errs.New` instead of `errs.Wrap(fmt.Errorf(...))` if you want to create an error with stack trace.

Reply:
I'm not sure from where this come from, but good spot!